### ISSUE TITLE
Demo app: menu icon is not inside of a button

### ISSUE THREAD

**Issue by hansl** [angular/material2#418]
The md-icon in the toolbar should be focusable and clickable.

**Comment by jelbourn** [comment]
Icons shouldn't be interactable, it should instead be the content of a button.
